### PR TITLE
Added initial version of Arm Forge docs

### PR DESCRIPTION
**Pull request by claoidek**

### CODE REVIEW

**Comment by kevinstratford** [comment]
Please leave this with me.

**Review by kevinstratford — DISMISSED** [review]
I think this is ready to go. I could probably do with someone giving it a go
(particularly if not used `ddt` before).

File: docs/data-tools/arm-forge.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+ARCHER2 has a license for up to 64 nodes (8192 cores) shared between
+all users at any one time. (Note cores are counted by the license, not
+MPI processes, threads, or any other software entity.)
+
+There are two ways of running the Arm user interface. If you have a good
+internet connection to ARCHER2, the GUI can be run on the front-end (with
+an X-connection).
+Alternatively, one can download a copy of the Forge remote client to your
+laptop or desktop, and run it locally. The remote client may be the
+preferred method for many users.
```
Comment (aturner-epcc) [inline docs/data-tools/arm-forge.md:17]: Be a bit stronger here: e.g. "We strongly recommend that users run the remote client on their laptop/desktop.

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Looks good - minor change and spotted a typo

**Review by kevinstratford — APPROVED** [review]

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
The images need to be updated to match Arm Forge 22.0.2 - this version has been installed to address a security vulnerability in 21.0.2

**Comment by aturner-epcc** [comment]
I have updated the documentation text. @claoidek Are you able to update the images of the interface? Once this is done, we should be good to make this live.

**Comment by claoidek** [comment]
@aturner-epcc Images have now been updated

File: docs/data-tools/arm-forge.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+then the "Change.." button. Choose the "SLURM (generic)" implementation
+from the MPI Implementation pull-down menu section and click "OK". You
+can then specify the required number of nodes/processes and so on.
+
+Click the OpenMP checkbox and select the relevant number of threads
+(if there is no OpenMP in the application itself, select 1 thread).
+
+Click the "Submit to Queue" checkbox and then the associated "Configure"
+button. A new dialogue of options will pop-up. In the top dialgoue box
+"Submission template file" enter `${FORGE_ROOT}/templates/archer2.qtf`
```
Comment (aturner-epcc) [inline docs/data-tools/arm-forge.md:147]: This file does not seem to exist?

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Thanks @claoidek Just one minor comment - there seems to be a template file missing.

File: docs/data-tools/arm-forge.md (hunk 2 above)
Comment (claoidek) [reply docs/data-tools/arm-forge.md:147]: @aturner-epcc This has been rectified

**Pull request merged**